src/dockerflow/django/middleware.py

```diff
@@ -53,9 +53,11 @@ def _build_extra_meta(self, request):
                           request.user.pk or '')
         if hasattr(request, '_id'):
             out['rid'] = request._id
-        if hasattr(request, '_logging_start_dt'):
-            td = datetime.datetime.utcnow() - request._logging_start_dt
-            out['t'] = int(td.total_seconds() * 1000)  # in ms
+        if hasattr(request, '_logging_start_timestamp'):
+            # timestamp it took, in milliseconds
```

Comment:
I don't think "timestamp" is the right word to use here. Maybe "duration of request, in milliseconds"?